Scripts/Source/minai_Sex.psc

```diff
@@ -247,7 +247,7 @@ Function StartSexOrSwitchToGroup(actor[] actors, actor akSpeaker, string tags=""
           OThread.StartAutoMode(ActiveOstimThreadID)
         EndIf
       EndIf
-      main.RegisterEvent(Main.GetActorName(akSpeaker) + " attempted to change the OStim scene to " + tags + " instead: " + newScene, "info_sexscene")
+      main.RegisterEvent(Main.GetActorName(akSpeaker) + " changed the OStim scene to " + tags + " instead: " + newScene, "info_sexscene")
```

Comment:
I know this was an issue on the old version too, but the LLM isn't going to know what "OStim scene" means. We should make this more descriptive.

Reply:
changed OStim to 'sex scene'